### PR TITLE
[stable/cratedb] - helm chart for the CrateDB

### PR DESCRIPTION
**Pull request by hmdmph**
Signed-off-by: Manoj Prasanna Handapangoda <hmdmph@yahoo.com>

#### What this PR does / why we need it:
**Helm chart for CrateDB** 
This is the initial implementation of the helm chart for the minimal version of the database CrateDB which is a distributed SQL database built on top of a NoSQL foundation. This is will enable users to create the CrateDB using helm chart easy and configurable way.

#### Special notes for your reviewer: 
This is the first implementation of the chart for CrateDB.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

**Comment by maorfr** [comment]
hey @hmdmph 

Have you considered hosting this chart and adding it to helm hub?
https://github.com/helm/hub/blob/master/Repositories.md

File: incubator/cratedb/OWNERS
Comment (cpanato) [inline incubator/cratedb/OWNERS]: new line

File: incubator/cratedb/templates/service.yaml
Comment (cpanato) [inline incubator/cratedb/templates/service.yaml]: use the new labels please

**Review by cpanato — CHANGES_REQUESTED** [review]
use the new labels and move to stable folder thanks!

**Comment by hmdmph** [comment]
Hi @maorfr,
> Have you considered hosting this chart and adding it to helm hub?
> https://github.com/helm/hub/blob/master/Repositories.md
 as of now no, You mean deploy this chart in some other repo(like github pages)?

File: incubator/cratedb/templates/service.yaml
Comment (hmdmph) [reply incubator/cratedb/templates/service.yaml]: new label added

File: stable/cratedb/.helmignore
Comment (cpanato) [inline stable/cratedb/.helmignore:23]: Please add the `OWNERS` file here as well

File: stable/cratedb/Chart.yaml
Comment (cpanato) [inline stable/cratedb/Chart.yaml]: bump to 1.0.0 since this will be in the stable folder

File: stable/cratedb/templates/internal-service.yaml
Comment (cpanato) [inline stable/cratedb/templates/internal-service.yaml]: new line here

Comment (cpanato) [inline stable/cratedb/templates/internal-service.yaml]: this label name is not correct, use `app.kubernetes.io/name` instead of `app` 
please see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#service-selectors

File: stable/cratedb/templates/service.yaml
Comment (cpanato) [inline stable/cratedb/templates/service.yaml:21]: missing the instance label, see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#service-selectors

File: stable/cratedb/templates/statefulset.yaml
Comment (cpanato) [inline stable/cratedb/templates/statefulset.yaml:21]: please see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#deployments-statefulsets-daemonsets-selectors

**Review by cpanato — CHANGES_REQUESTED** [review]
see my comments, thanks for this PR :)

**Comment by dart-x** [comment]
> see my comments, thanks for this PR :)
@cpanato Addressed PR comments.

**Comment by hmdmph** [comment]
> see my comments, thanks for this PR :)

@cpanato addressed your PR comments. Thanks.

**Comment by maorfr** [comment]
Github pages is a great way to host your own charts!
This will give you more freedom around management, and you will not be blocked by us (maintainers).

This means that when users want to install your chart, instead of installing them from stable, they will install it from (for example) mycompany-charts:

$ helm install --name my-release mycompany-charts/mychart
Here are some instructions about how to host your own charts repo:
https://github.com/helm/hub/blob/master/docs/github.md#github-pages-as-a-repository

In order for people to find out about your charts repo, you should add it to helm hub:
https://hub.helm.sh/

Here are some instructions about how to add your charts repo to helm hub:
https://github.com/helm/hub#how-to-add-your-helm-charts

**Pull request closed without merging**
